UI/EncyclopediaDetailPanel.cpp

```diff
@@ -1397,11 +1397,17 @@ namespace {
             return;
         }
         else if (item_name == "ENC_GAME_RULES") {
-            for (auto& [rule_name, rule] : GetGameRules()) {
-                if (rule.ValueIsDefault())
-                    detailed_description += UserString(rule_name) + " : " + rule.ValueToString() + "\n";
+            std::string last_category;
+            detailed_description += UserString("GENERAL") + ":\n";
+            for (const auto& rule : GetGameRules().GetSortedByCategoryAndRank()) {
```

Comment:
```suggestion
            for (const auto* rule : GetGameRules().GetSortedByCategoryAndRank()) {
```

Reply:
ok then that same change should be in GalaxySetupWnd.cpp as well.. so you get a copy of a pointer instead of reference to a pointer but I guess that's preferable / less confusing